config/webhook/service.yaml

```diff
@@ -0,0 +1,20 @@
+
+apiVersion: v1
```

Comment:
starting with an empty line? Maybe include the yaml-start-of-document `---` instead (and in other yaml files)

Reply:
again this is auto generated file